### DISCUSSION TITLE
SQLiteStudio在mac上的開啟方法

### DISCUSSION THREAD

**Discussion by Yunhans** [1101-GS3073-C/Python-Community-GS3073-C#44] [General]
滑鼠左鍵雙擊兩下是打不開的，會顯示“SQLiteStudio: cannot be opened because the developer cannot be verified.”，要按滑鼠右鍵再選擇打開就行了，至少我的是如此

**Comment by chia-kai-chang** [comment]
我沒有Mac,印象中要先按兩下sqlitestudio,他會說不讓你開。

然後點選
![image](https://user-images.githubusercontent.com/1768891/136663036-fedd311c-1f8f-45ff-a5d8-864dcbfbca58.jpeg)

注意左下角的鎖頭，應該可以開啟權限，之後就可以開啟sqlitestudio了。
